core/test/unit/services/auth/api-key/admin_spec.js

```diff
@@ -0,0 +1,197 @@
+const authenticateAdminApiKey = require('../../../../../server/services/auth/api-key/admin');
+const common = require('../../../../../server/lib/common');
+const jwt = require('jsonwebtoken');
+const models = require('../../../../../server/models');
+const should = require('should');
+const sinon = require('sinon');
+const testUtils = require('../../../../utils');
+
+const sandbox = sinon.sandbox.create();
+
+describe('Admin API Key Auth', function () {
+    before(models.init);
+    before(testUtils.teardown);
+
+    beforeEach(function () {
+        const fakeApiKey = {
+            id: '1234',
+            type: 'admin',
+            secret: Buffer.from('testing').toString('hex'),
+            get(prop) {
+                return this[prop];
+            }
+        };
+        this.fakeApiKey = fakeApiKey;
+        this.secret = Buffer.from(fakeApiKey.secret, 'hex');
+
+        this.apiKeyStub = sandbox.stub(models.ApiKey, 'findOne');
+        this.apiKeyStub.returns(new Promise.resolve());
```

Comment:
I don't think there should be a `new` keyword here or below :relaxed:

Reply:
👍 I was following a number of other places in the codebase that do it this way, maybe a case for a separate PR with a global find/replace

Reply:
Ah fair enough - yeah :+1: